hibernate-core/src/main/java/org/hibernate/engine/spi/SessionDelegatorBaseImpl.java

```diff
@@ -1163,7 +1163,7 @@ public Integer getJdbcBatchSize() {
 	}
 
 	@Override
-	public void setJdbcBatchSize(int jdbcBatchSize) {
+	public void setJdbcBatchSize(Integer jdbcBatchSize) {
```

Comment:
What is the intent of changing the type here?  The only point of having Integer for the state field was to be able to know whether a value has been set or not (null).  Why is passing null here valid?